cpp/oneapi/dal/algo/decision_forest/detail/infer_ops_dpc.cpp

```diff
@@ -29,9 +29,9 @@ struct infer_ops_dispatcher<data_parallel_policy, Float, Task, Method> {
     infer_result<Task> operator()(const data_parallel_policy& ctx,
```

Comment:
Can this whole structure be reused. I mean this `operator()` is common for many algorithms

Reply:
It is common now, there will be differences once we start adding distributed algorithms